src/commands/serviceConnector/ContainerPickStep.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem, nonNullValue } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../utils/localize";
+import { IServiceConnectorContext } from "./IServiceConnectorContext";
+
+export class ContainerPickStep extends AzureWizardPromptStep<IServiceConnectorContext> {
```

Comment:
Generally this is called a `ListStep` not a `PickStep`. 

Though I get that there is already a `ContainerListStep`, I think we should rename things to be more explicit. The `ContainerListStep` today is technically the `ReplicaContainerListStep` and has a hard requirement of having a `replica` defined on the context (which happens in the previous step). So maybe just rename that one and then call this `ContainerListStep` or `TemplateContainerListStep`?